src/monobase/monogen.py

```diff
@@ -22,9 +22,10 @@ def otel_attributes(self):
 
 
 # uv venv --seed does not install deprecated setuptools or wheel for Python 3.12
+# Packaging is needed for flash-attn, etc.
 # Explicitly declare them here
 # Versions are not pinned and we will use whatever Torch index has
-SEED_PKGS = ['pip', 'setuptools', 'wheel']
+SEED_PKGS = ['pip', 'packaging', 'setuptools', 'wheel']
```

Comment:
Just realized that this won't affect existing generations unless we roll a new one out. But we don't need to since we never `uv pip install` into user venv in prod. So this addition is only needed in mini mono when we build image locally with `cog build --x-fast`.

Reply:
I was wrong. Seed packages are also frozen into requirements so I need to update them.